### PR TITLE
fix(#109): Changes required in the home-spa code as the dependencies need to be upgraded 

### LINKED ISSUE #109: Changes required in the home-spa code as the dependencies need to be upgraded
## Describe the bug:
https://github.com/1-Platform/one-platform/pull/108
To accommodate the breaking changes, home-spa minor code changes are required

## Steps to reproduce:
- Update package.json as https://github.com/1-Platform/one-platform/pull/108 mentioned
- run `npm run dev`
- Open browser
- Go to http://localhost:1234
- Open console
- Check for the errors

## Screenshots:
Screenshot 2020-03-23 at 7 58 50 PM

## Environment;

- Operating System(s): Linux/macOS/Windows [Variant and Version]
- Browser: Firefox/Chrome/Safari [Version] / Other (please specify)

### PR DESCRIPTION
**Pull request by hybridx**
Dependency updated html-webpack-plugin tp 4.0.0 and supporting code fix

> ### Resolves #109 

## Explain the feature/fix:
#108
To accommodate the breaking changes, home-spa minor code changes were required

## Does this PR introduce a breaking change?
The version for html-webpack-plugin is now `4.0.0`
> No (not for any other modules)

#### All systems are go!
Screenshot 2020-03-23 at 8 40 25 PM

### Ready-for-merge Checklist

- [X] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [X] Does the change have appropriate unit tests?
- [X] Did tests pass?

### CODE REVIEW

File: packages/home-spa/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "css-loader": "3.4.2",
     "file-loader": "6.0.0",
     "html-loader": "1.0.0",
-    "html-webpack-plugin": "3.2.0",
+    "html-webpack-plugin": "4.0.0",
```
Comment (deshmukhmayur) [inline packages/home-spa/package.json]: Can you make it up to `^4.0.0`

So it will download the latest minor version whenever available.

**Comment by deshmukhmayur** [comment]
And also, mention the version bump in your description.

File: packages/home-spa/package.json (hunk 1 above)
Comment (hybridx) [reply packages/home-spa/package.json]: I will do that, but the renovate bot will make a PR to pin the dependency @deshmukhmayur

Comment (deshmukhmayur) [reply packages/home-spa/package.json]: We can discard that PR. We don't need multiple PR for the same fix.

Comment (hybridx) [reply packages/home-spa/package.json]: Then we will have to discard this PR https://github.com/1-Platform/one-platform/pull/117

**Pull request merged**